src/index.js

```diff
@@ -1 +1,37 @@
+/* eslint-disable no-console */
 'use strict';
+import 'dotenv/config';
+import express from 'express';
+import cors from 'cors';
+import { authRouter } from './routes/auth.routes.js';
+import { errorMiddleware } from './middlewares/errorMiddleware.js';
+import cookieParser from 'cookie-parser';
+import { userRouter } from './routes/user.routes.js';
+import { ConsoleLoger } from './untils/consoleLoger.js';
```

Comment:
There is a typo in 'ConsoleLoger'. It should be 'ConsoleLogger'. Ensure that the import statement and all usages are corrected.